pyrsa/data/dataset.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Definition of RSA Dataset class and subclasses
+@author: baihan, jdiedrichsen
+"""
+
+import numpy as np
+import pyrsa as rsa
+
+
+class DatasetBase:
```

Comment:
Yes, that's true, but since I accidentally start my branch rdm-dev from the data-dev, so I have to manually update it in rdm-dev from data-dev, before pull #16 is approved and merged.

Reply:
I'm sorry I don't think I understand. Why's that? Are the dataset classes imported by the rdm modules?